### DISCUSSION TITLE
大家好

### DISCUSSION THREAD

**Discussion by passbyDFX** [share121/inter-knot#4982] [其他]
新来的，内啥，怎么换头像

**Comment by passbyDFX** [comment]
0.o

**Comment by yyyzmo** [comment]
插眼

**Comment by yinyangyehen** [comment]
➕1

**Comment by cantonesefish** [comment]
在github里，点进自己的头像，在里面找找，实在不行开个翻译

**Comment by HLXS-HLXS** [comment]
GitHub官网左上角点头像，点设置，点profile picture【个人资料图片】下面的edit【编辑】，点upload a photo…【上传图片】，选择图片双击上传，图片不能大于1mb

**Reply by passbyDFX** [reply]
非常感谢！

**Comment by Pardo-Felis-1** [comment]
这委托咱接了
![img_1709975317_1717922490177](https://github.com/user-attachments/assets/87d2b041-272f-4ad3-800e-91cafba047c7)

**Reply by passbyDFX** [reply]
哇酷哇酷，帕朵！
